src/Files.App/DataModels/NavigationControlItems/DriveItem.cs

```diff
@@ -224,7 +224,14 @@ public async Task LoadDriveIcon()
 			if (IconData is null)
 			{
 				if (!string.IsNullOrEmpty(DeviceID))
+				{
+					if (string.Equals(DeviceID, "network-folder"))
```

Comment:
I'm not a huge fan of having three imbricated `if`s like this. Even if it's a detail, it grows the cognitive complexity by quite a margin. 

Maybe we could remove a level of nesting by inverting the first `if` like this at the beginning of the method?

```
if (IconData is not null)
{
	Icon = await IconData.ToBitmapAsync();
	return;
}
```